docs/src/main/paradox/index.md

```diff
@@ -19,7 +23,15 @@ Based on a protobuf service definition, akka-grpc can generate:
 * On the server side, code to create an Akka HTTP route based on your implementation of the API
 * On the client side, a client for the API.
 
+### gRPC vs REST
+
+* Where REST is more flexible about transport and encoding, gRPC standardizes on HTTP/2 and Protobuf.
```

Comment:
Huh, this I never heard before, I thought it was very tightly coupled with the GET, POST, UPDATE, DELETE methods of HTTP?

Reply:
That's true, perhaps let's just say `Where REST is more flexible about encoding`